### ISSUE TITLE
Error while sending packet

### ISSUE THREAD

**Issue by tomasell** [madchicken/homebridge-comelit-hub#32]
Hi,
I've more than 100 Comelit objects, so I need to run multiple Homebridge instances.
I've configured the first one scanning only lights and the second one scanning others sensors.

I'm using `hb-service` to run more than one instance, and these are the configurations:

```json
{
    "bridge": {
        "name": "Homebridge AEA8",
        "username": "0E:97:16:D5:AE:A8",
        "port": 51193,
        "pin": "###-###-###"
    },
    "accessories": [],
    "platforms": [
        {
            "name": "Config",
            "port": 8581,
            "auth": "form",
            "theme": "indigo",
            "tempUnits": "c",
            "lang": "it",
            "sudo": true,
            "platform": "config"
        },
        {
            "name": "Comelit",
            "broker_url": "###.###.###.###",
            "username": "###",
            "password": "###",
            "blind_closing_time": 35,
            "keep_alive": 120000,
            "avoid_duplicates": true,
            "hide_lights": false,
            "hide_blinds": true,
            "hide_thermostats": true,
            "hide_power_suppliers": true,
            "hide_outlets": true,
            "hide_others": true,
            "platform": "Comelit"
        }
    ]
}
```

```json
{
    "bridge": {
        "name": "Homebridge FAA9",
        "username": "0E:A2:E5:95:FA:A9",
        "port": 51716,
        "pin": "###-###-###"
    },
    "accessories": [],
    "platforms": [
        {
            "name": "Blinds",
            "port": 8582,
            "auth": "form",
            "theme": "green",
            "tempUnits": "c",
            "lang": "it",
            "sudo": true,
            "platform": "config"
        },
        {
            "name": "Comelit Blinds",
            "broker_url": "###.###.###.###",
            "username": "###",
            "password": "###",
            "blind_closing_time": 35,
            "keep_alive": 120000,
            "avoid_duplicates": true,
            "hide_lights": true,
            "hide_blinds": false,
            "hide_thermostats": false,
            "hide_power_suppliers": false,
            "hide_outlets": false,
            "hide_others": false,
            "platform": "Comelit"
        }
    ]
}
```

When I run the second one, it doesn't discover anything and you can see this error message:
```
[08/12/2020, 15:43:07] [Comelit Blinds] Error while sending packet
(node:387) UnhandledPromiseRejectionWarning: Error: Timeout for message:  {"req_type":0,"seq_id":4,"req_sub_type":-1,"sessiontoken":"648829150","obj_id":"GEN#17#13#1","detail_level":1}
    at /usr/local/lib/node_modules/homebridge-comelit-platform/node_modules/comelit-client/src/promise-queue.ts:49:30
    at Array.reduce (<anonymous>)
    at ComelitClient.cleanPending (/usr/local/lib/node_modules/homebridge-comelit-platform/node_modules/comelit-client/src/promise-queue.ts:43:40)
    at Timeout._onTimeout (/usr/local/lib/node_modules/homebridge-comelit-platform/node_modules/comelit-client/src/promise-queue.ts:36:14)
    at listOnTimeout (internal/timers.js:554:17)
    at processTimers (internal/timers.js:497:7)
(Use `node --trace-warnings ...` to show where the warning was created)
(node:387) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
```

**Comment by madchicken** [comment]
Are they running on two different hosts?

**Comment by tomasell** [comment]
Both Homebridge servers are running on the same host ( a Raspberry ). As you can see, I'd like to run two  servers with different plugin configuration.

**Comment by madchicken** [comment]
This is a configuration I never tried. I'll do a test here in my environment

**Comment by tomasell** [comment]
Hi @madchicken , 
do you have any updates about this issue? 
Thanks!

**Comment by madchicken** [comment]
Sorry, I still need to setup a test here. I didn't have time in the last few weeks. Please stay tuned

**Comment by madchicken** [comment]
I was reading the code, what I suspect is that the generation of the topic ID is somehow wrong.
Requests: 

- provide please logs of both installation after a restart.
- try manually setting `clientId` in the configuration. Is a string and must be different between the two instances

**Comment by madchicken** [comment]
I just published a new version. Please, try with the new one

**Comment by tomasell** [comment]
Resolved with v2.5.1. Thanks @madchicken !
